### DISCUSSION TITLE
How to access beta channel of the language packs

### DISCUSSION THREAD

**Discussion by Pippo8291** [AnySoftKeyboard/AnySoftKeyboard#3350] [Q&A] [answered]
Hi, is there any trick to access the beta Channel for the language packs?

Everything worked out for the Main app but for the addons it always says: "Your account isn't currently eligible for this app's testing program."

Thanks for any help

**Comment by menny** [comment, marked as answer]
Unfortunately, no. I need to manually fix these issues.

Please open a ticket with the pack that fails to allow beta testing, and I'll fix that.

**Reply by Pippo8291** [reply]
thanks for your reply... i just closed my issue from the weekend as long as i found a workaround by directly loading the version from the deployments.
